openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx

```diff
@@ -213,23 +229,26 @@ const CustomNode = (props: NodeProps) => {
       ) : null}
       <section
         className={classNames('tw--mx-3 tw-px-3', {
-          'tw-h-36 tw-overflow-y-auto': columns?.length,
+          // 'tw-h-36 tw-overflow-y-auto': !isEmpty(columns),
         })}
         id="table-columns">
         <div className="tw-flex tw-flex-col tw-gap-y-1 tw-relative">
-          {columns?.map(
-            (c: { name: string; constraint: string }, i: number) => (
-              <Fragment key={i}>
-                <div
-                  className="tw-p-1 tw-rounded tw-border tw-text-grey-body"
-                  data-testid="column"
-                  key={c.name}>
-                  {getConstraintIcon(c.constraint, 'tw-')}
-                  {c.name}
-                </div>
-              </Fragment>
-            )
-          )}
+          {(Object.values(columns || {}) as Column[])?.map((c, i) => (
+            <Fragment key={i}>
+              <div
+                className="tw-p-1 tw-rounded tw-border tw-text-grey-body tw-relative"
+                data-testid="column">
+                {getHandle(
+                  type,
+                  isConnectable,
+                  isNewNode,
+                  c.fullyQualifiedName
+                )}
+                {getConstraintIcon(c.constraint, 'tw-')}
+                {c.name}
+              </div>
+            </Fragment>
```

Comment:
```suggestion
              <div
              key={i}
                className="tw-p-1 tw-rounded tw-border tw-text-grey-body tw-relative"
                data-testid="column">
                {getHandle(
                  type,
                  isConnectable,
                  isNewNode,
                  c.fullyQualifiedName
                )}
                {getConstraintIcon(c.constraint, 'tw-')}
                {c.name}
              </div>
```

Fragment is not required here

Reply:
Thanks for catching 👍